src/dialogs/examples/basic/O365ConnectorCardActionsDialogV1.ts

```diff
@@ -0,0 +1,206 @@
+                                    .style("compact")
+                                    .choices([
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Bot").value("Bot"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Tab").value("Tab"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Connector").value("Connector"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Compose Extension").value("Compose Extension"),
+                                    ]),
+                            ])
+                            .actions([
+                                new teams.O365ConnectorCardHttpPOST(session)
+                                    .id("cardAction-1-btn-1")
+                                    .name(Strings.send)
+                                    .body(JSON.stringify({
+                                        list1: "{{list-1.value}}",
+                                        list2: "{{list-2.value}}",
+                                        list3: "{{list-3.value}}",
+                                        list4: "{{list-4.value}}",
+                                    })),
+                            ]);
+
+        let section1    = new teams.O365ConnectorCardSection(session)
+                            .markdown(true)
+                            .title(Strings.hey_there)
+                            .potentialAction([cardAction1]);
+
+        let cardAction2 = new teams.O365ConnectorCardActionCard(session)
+                            .id("CardsTypeSection2")
+                            .name(Strings.multiple_choice)
+                            .inputs([
+                                new teams.O365ConnectorCardMultichoiceInput(session)
+                                    .id("list-5")
+                                    .title(Strings.combo_box_title)
+                                    .isMultiSelect(true)
+                                    .isRequired(false)
+                                    .style("compact")
+                                    .choices([
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Hero Card").value("Hero Card"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Thumbnail Card").value("Thumbnail Card"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("O365 Connector Card").value("O365 Connector Card"),
+                                    ]),
+                                new teams.O365ConnectorCardMultichoiceInput(session)
+                                    .id("list-6")
+                                    .title(Strings.pick_a_time)
```

Comment:
Is this correct? "Pick a time" doesn't seem to match the provided choices.

Reply:
Done.